pretyping/pretyping.mli

```diff
@@ -32,12 +32,17 @@ val clear_bidirectionality_hint : GlobRef.t -> unit
 
 (** An auxiliary function for searching for fixpoint guard indexes *)
 
+type possible_guard =
+  bool (* true if can be a cofixpoint *) *
+  int list list (* list of non-empty lists of possible indices if can be a fixpoint *)
```

Comment:
Maybe this could be a record?
Also it feels strange that we confuse "maybe cofix, maybe fix" and "known cofix".

Reply:
I agree to both remarks... I'll think about them.

Reply:
>I agree to both remarks... I'll think about them.


any news?

Reply:
I used a record. Regarding "known cofix" it is a priori never the case, that is, it has to be a cofix if there is no possible fix indices, but we don't know for sure until it is checked.